extensions/adapters/vector/src/main/java/mil/nga/giat/geowave/adapter/vector/index/ChooseBestMatchIndexQueryStrategy.java

```diff
@@ -68,6 +69,10 @@ public boolean hasNext() {
 						nextIdx = (PrimaryIndex) nextChoosenIdx;
 						if (nextIdx.getIndexStrategy().getOrderedDimensionDefinitions().length == 0) continue;
 						final List<MultiDimensionalNumericData> constraints = query.getIndexConstraints(nextIdx.getIndexStrategy());
+
```

Comment:
Choose best match only uses stats.
Some indices do not have stats (liek NullIndex)